ttcp.sh

```diff
@@ -158,6 +158,14 @@ __ttcp::is_env_ok () {
     return 0
 }
 
+__ttcp::encrypt () {
```

Comment:
`encrypt` seems not appropriate name as this part does more than encryption. How about `encode` ?